core/parachain/approval/approval_distribution.cpp

```diff
@@ -619,7 +619,8 @@ namespace kagome::parachain {
 
     metrics_registry_->registerCounterFamily(
         kMetricNoShowsTotal,
-        "Number of assignments which became no-shows in the approval voting subsystem");
+        "Number of assignments which became no-shows in the approval voting "
+        "subsystem");
```

Comment:
I as much of us prefer to avoid jumbo line-comment, and `clang-format` helps to wrap them automatically. If result comment looks ugly by some developer opinion, he can edit it by himself or to off formatting in-place manually.